stories/organisms/Dialog.stories.tsx

```diff
@@ -0,0 +1,191 @@
+import * as React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Alert, Button, Dialog, Confirm } from '@textkernel/oneui';
+
+const meta: Meta<typeof Dialog> = {
+    title: 'Organisms/Dialogs',
+    component: Dialog,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    subcomponents: { Alert, Confirm } as any,
+};
+
+export default meta;
+
+type DialogStory = StoryObj<typeof Dialog>;
+
+export const _Dialog: DialogStory = {
+    name: 'Dialog',
+    args: {
```

Comment:
In storybook, it is hard to set the variant. It asks for some JSON string instead of offering some radio buttons. Can you check?

Reply:
I was able to make it to string. It doesn't work with radio buttons because the type is Omit<...> and not ButtonContext directly.